### ISSUE TITLE
replace hyphen and supercript sign with SpaceCharacter

### ISSUE THREAD

**Issue by ilomedia** [dmitryd/typo3-realurl#292]
Hi,

in french (and maybe other languages) hyphen (‘) and superscript (') signs are often used as word separator. As for now, realurl removes them and the url contains unwanted concatenated words.
Couls you add those character to the convertToSafeString method?

For the moment i extended \DmitryDulepov\Realurl\Utility::convertToSafeString and added those two lines but this is not ideal in case of updates : 

`$processedTitle = str_replace("'", $spaceCharacter, $processedTitle);`
`$processedTitle = str_replace("’", $spaceCharacter, $processedTitle);`

NB : post processing can not be done since characters are removed before

thanks

**Comment by dmitryd** [comment]
Yes, may affect existing URLs... On the other hand, if entries are in the cache already, they will be used and pages will not show 404s.

**Comment by dmitryd** [comment]
The change will go to 2.2.0 I think. I plan to release 2.1.5 and than go for 2.2.0.

**Comment by ilomedia** [comment]
thanks

**Comment by sruegg** [comment]
Hi,

Thanks for this change! Same here for me for french page titles.

Perhaps changing the empty replacement by the $spaceCharacter is enough? (saying that without knowing other implications ;-) )
https://github.com/dmitryd/typo3-realurl/blob/development/Classes/Utility.php#L101

Thanks a lot for this awesome ext by the way!

**Comment by ilomedia** [comment]
Hi,

any news on this? does not seem to be in 2.2.1 :(
